apps/nestjs-api/src/reminder-emails/reminder-emails.service.ts

```diff
@@ -366,100 +499,152 @@ export class ReminderEmailsService {
+      /{{{Recipient.FirstName}}}/g,
+      `${firstName}`
+    )
 
-  //   const params = {
-  //     Destination: {
-  //       ToAddresses: isProductionOrDemonstration()
-  //         ? ['anilakarsu93@gmail.com']
-  //         : [this.config.get('NX_DEV_MODE_EMAIL_RECIPIENT')],
-  //     },
-  //     Message: {
-  //       Body: {
-  //         Html: { Data: sanitizedHtml },
-  //       },
-  //       Subject: { Data: template.Subject },
-  //     },
-  //     Source: 'career@redi-school.org',
-  //   }
+    const params = {
+      Destination: {
+        ToAddresses: isProductionOrDemonstration()
+          ? [email]
+          : [this.config.get('NX_DEV_MODE_EMAIL_RECIPIENT')],
+        BccAddresses: [SENDER_EMAIL],
+      },
+      Message: {
+        Body: {
+          Html: { Data: sanitizedHtml },
+        },
+        Subject: { Data: template.Subject },
+      },
+      Source: `${SENDER_NAME} <${SENDER_EMAIL}>`,
+    }
 
-  //   this.ses.sendEmail(params, (err, data) => {
-  //     if (err) console.log(err, err.stack)
-  //     else console.log(data)
-  //   })
+    this.ses.sendEmail(params, (err, data) => {
+      if (err) console.log(err, err.stack)
+      else console.log(data)
+    })
 
-  //   return { message: 'Email sent' }
-  // }
+    return { message: 'Email sent' }
+  }
 
-  // async sendLogMentoringSessionsReminder({
-  //   email,
-  //   firstName,
-  //   userType,
-  //   mentorshipMatchAgeInDays,
-  // }) {
-  //   const sfEmailTemplateDeveloperName =
-  //     userType === UserType.MENTOR
-  //       ? // Mentor Emails for 2 and 4 weeks
-  //         mentorshipMatchAgeInDays === 14
-  //         ? 'Mentor_Log_Mentoring_Sessions_Reminder_1_1711110740940'
-  //         : 'Mentor_Log_Mentoring_Sessions_Reminder_2_1711112496532'
-  //       : // Mentee Emails for 2 and 4 weeks
-  //       mentorshipMatchAgeInDays === 14
-  //       ? 'Mentee_Log_Mentoring_Sessions_Reminder_1_1711114670729'
-  //       : 'Mentee_Log_Mentoring_Sessions_Reminder_2_1711115347143'
-
-  //   const template = await this.emailTemplatesService.getEmailTemplate(
-  //     sfEmailTemplateDeveloperName
-  //   )
-
-  //   if (!template) {
-  //     throw new Error(
-  //       `Email template not found: ${sfEmailTemplateDeveloperName}`
-  //     )
-  //   }
+  async sendLogMentoringSessionsReminder({
+    email,
+    firstName,
+    userType,
+    mentorshipMatchAgeInDays,
+  }) {
+    const sfEmailTemplateDeveloperName =
+      userType === UserType.MENTOR
+        ? // Mentor Emails for 2 and 4 weeks
+          mentorshipMatchAgeInDays === 14
+          ? 'Mentor_Log_Mentoring_Sessions_Reminder_1_1711110740940'
+          : 'Mentor_Log_Mentoring_Sessions_Reminder_2_1711112496532'
+        : // Mentee Emails for 2 and 4 weeks
+        mentorshipMatchAgeInDays === 14
+        ? 'Mentee_Log_Mentoring_Sessions_Reminder_1_1711114670729'
+        : 'Mentee_Log_Mentoring_Sessions_Reminder_2_1711115347143'
```

Comment:
Shouldn't the templates in these conditions switch places to

```
mentorshipMatchAgeInDays === 14
          ? 'Mentor_Log_Mentoring_Sessions_Reminder_2_1711112496532'
          : 'Mentor_Log_Mentoring_Sessions_Reminder_1_1711110740940'
```
?

Or perhaps creating variables for each template would be a good idea here?

Reply:
Not really, as they're only used once. Even if we create a variable, we will use that variable only once again, which I'm not sure how it's gonna help. What do you think?

Reply:
I meant the template name, not the template. Without variables is also fine. But the template names need to switch places here. 

Do you agree or I misunderstood the logic?

Reply:
Ah good catch, you're right. They're in incorrect order. Very very nice catch.

Reply:
No wait, the rule is 2 Weeks - 4 Weeks. So 14 is correct there. I will add a comment to avoid confusion later though.

Reply:
You are right! Sorry for the misunderstanding.